ebu_tt_live/node/delay.py

```diff
@@ -0,0 +1,71 @@
+
+from .base import Node
+from datetime import timedelta
+from ebu_tt_live.bindings._ebuttdt import LimitedClockTimingType, FullClockTimingType
+import time
+import random
+
+
+class FixedDelayNode(Node):
+
+    _reference_clock = None
+    _sequence = None
+
+    def __init__(self, node_id, carriage_impl, reference_clock):
+        super(FixedDelayNode, self).__init__(node_id, carriage_impl)
+        self._reference_clock = reference_clock
+
+        random_int = random.randint(0, 9999999999)
+        new_identifier = 'sequence_{0}'.format(str(random_int))
```

Comment:
Might be nice to allow a desired sequence ID to be specified in the constructor. Also the delay period.